frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -125,6 +125,13 @@ export function doWithToken<T>(fn: (token: string) => Promise<T>, tries: number
   });
 }
 
+export function trackAnalytics(eventType: string, eventProperties: Object): void {
+  Request.sendJSONReceiveJSON(`/api/analytics/${eventType}`, {
```

Comment:
```suggestion
  // Note that the Promise from sendJSONReceiveJSON is not awaited or returned here, since
  // failing analytics events should not have an impact on the application logic.
  Request.sendJSONReceiveJSON(`/api/analytics/${eventType}`, {
```